update.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# CHECK DISK FREE SPACE
+required_disk_free=200000
+disk_free=$(df -l | grep '/dev/root' | awk '{print $4}')
+echo "FREE DISK SPACE: $disk_free M"
+if [ "$disk_free" -lt "$required_disk_free" ]; then
+  echo "error > NOT ENOUGH FREE DISK SPACE"
```

Comment:
consider echoing space required?

    NOT ENOUGH FREE DISK SPACE (200M required)